cmd/cluster/stage/stage.go

```diff
@@ -64,6 +72,72 @@ func NewCmd() *cobra.Command {
 
 // RunCmd runs the "ocne node update" command
 func RunCmd(cmd *cobra.Command) error {
+	// If the cluster name is not set, check for an existing
+	// cluster named "ocne".  If a cluster with that name is
+	// not in the cache, then treat the invocation as not
+	// using a cluster config file.
+	//
+	// This is a loop to access break, but should never
+	// execute more than once.
+	if clusterName == "" {
+		for clusterName == "" {
```

Comment:
Would be cleaner if it was a function call so it didn't have to do the for loop.

Reply:
Now that there is no longer a default cluster name, this code has been simplified and there is no need for the loop anymore.